index.html

```diff
@@ -966,13 +966,16 @@ <h2>Context Processing Algorithm</h2>
       <a>term definitions</a> which specify how
       keys and values have to be interpreted as well as the current <a>base IRI</a>,
       the <a>vocabulary mapping</a>, the <a>default language</a>,
-      <span class="changed">and an optional <dfn>previous context</dfn>,
+      <span class="changed">the <a>default base direction</a>,
+        and an optional <dfn>previous context</dfn>,
         used when a non-propagated <a>context</a> is defined</span>.
       Each <a>term definition</a> consists of:</p>
     <ul>
       <li>an <dfn>IRI mapping</dfn>,</li>
       <li>a boolean flag <dfn>reverse property</dfn>,</li>
-      <li>an optional <dfn>type mapping</dfn>, or <dfn>language mapping</dfn>,</li>
+       <li>an optional <dfn>type mapping</dfn>,</li>
+       <li>an optional <dfn>language mapping</dfn>,</li>
+      <li class="changed">an optional <dfn>direction mapping</dfn>,</li>
```

Comment:
(pardon this comment here, github review limitiation, and a bit off topic) Down a few lines the "protected" text looks like it's missing a word.

Reply:
And while I'm here... should these be using `<a>` that links to `<dfn>` text it terms.html (or elsewhere)?

Reply:
I changed that to `a <dfn>protected</dfn> flag`. This defines the values contained within a **term definition**, so this does seem to be the source of those definitions.

However, there may be some uses of `<a>protect</a>` which refer to the concept of protection.